components/Caretaker.js

```diff
@@ -30,7 +29,7 @@ const Caretaker = ({history, navigation}) => {
     })
   }
 
-  const [isVisible, setIsVisible] = useState(false);
+  const [modalVisible, setIsVisible] = useState(false);
```

Comment:
I know I've heard that for booleans, the prefix "is" is usually a proper technique. That being said, this makes sense to me, and I don't recommend changing it. Just a heads up for laters.

Reply:
I didn't know that, but it makes sense! If it's best practices and easy to change then I think that would be a good choice.